corehq/form_processor/models/cases.py

```diff
@@ -581,6 +581,52 @@ def host(self):
         result = self.get_parent(relationship=CommCareCaseIndex.EXTENSION)
         return result[0] if result else None
 
+    def save(self, *args, with_tracked_models=False, **kw):
```

Comment:
It's a less safe transformation, but might be nicer to flip the default of this argument to `True` so that `case.save()` would save tracked models by default. Code that currently calls `case.save()` may need to be changed to `case.save(with_tracked_models=False)`, depending on how much impact that has on save performance. Thoughts?